provider/terraform/resource_override.rb

```diff
@@ -16,9 +16,40 @@
 
 module Provider
   class Terraform < Provider::AbstractCore
-    # Terraform-specific overrides to api.yaml.
+    # Collection of properties allowed in the ResourceOverride section for
+    # Terraform. All properties should be `attr_reader :<property>`
+    module OverrideProperties
+    end
+
+    # A class to control overridden properties on terraform.yaml in lieu of
+    # values from api.yaml.
     class ResourceOverride < Provider::ResourceOverride
-      # TODO: Add Terraform specific properties here.
+      include OverrideProperties
+
+      def apply(resource)
+        unless description.nil?
```

Comment:
I think `update_overriden_properties` in Provider::ResourceOverride#apply will overwrite the description that was extended here. 

This means that if we have the following:

```
api.yaml description: foo
terraform.yaml description: {{description}}bar
```

After Provider::Terraform::ResourceOverride#apply, we have

```
api.yaml description: foobar
terraform.yaml description: {{description}}bar
```

Then `super` is called.
After Provider::ResourceOverride#apply, we will have:

```
api.yaml description: {{description}}bar
terraform.yaml description: {{description}}bar
```

Reply:
I think the actual execution is slightly different:

```
api.yaml description: foo
terraform.yaml description: {{description}}bar
```

After Provider::Terraform::ResourceOverride#apply, we have

```
api.yaml description: foobar
terraform.yaml description: foobarbar
```

The change is above. It is due to line 35.

Then when super gets called:

```
api.yaml description: foobarbar
terraform.yaml description: foobarbar
```